test/config.test1.yaml

```diff
@@ -227,7 +250,7 @@ solving:
     max_iterations: 6
 
   solver:
-    name: glpk
+    name: cbc
```

Comment:
was this necessary for the CI? If yes, do you know why?

Reply:
Yes but I don't know why yet. I think it somehow outperforms glpk. We might need to switch to Highs some time in the future for the CI